pkg/mbedtls/contrib/sha256_alt.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_mbedtls
+ *
+ * @{
+ * @file
+ *
+ * @author      Peter Kietzmann <peter.kietzmann@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include <assert.h>
+#include <string.h>
+
+#include "riot_config.h"
+
+#if defined(MBEDTLS_SHA256_ALT)
```

Comment:
```suggestion
#if IS_ACTIVE(MBEDTLS_SHA256_ALT)
```
(and add include of kernel_defines.h)